src/components/ListV2/ListHeader.tsx

```diff
@@ -0,0 +1,97 @@
+import styled from '@emotion/styled'
+import { HTMLProps, MouseEvent, ReactNode } from 'react'
+import Icon from '../Icon'
+
+const getAriaSort = (
+  sort: 'asc' | 'desc' | 'none',
+): HTMLProps<HTMLDivElement>['aria-sort'] => {
+  if (sort === 'asc') return 'ascending'
+
+  if (sort === 'desc') return 'descending'
+
+  return sort === 'none' ? 'none' : 'other'
+}
+
+const ArrowDownIcon = styled(Icon)``
+const ArrowUpIcon = styled(Icon)``
+
+const StyledIconContainer = styled.div`
+  display: flex;
+  flex-direction: column;
+  color: ${({ theme }) => theme.colors.neutral.textWeak};
+`
+
+const StyledTh = styled('div', {
```

Comment:
Might need to be renamed if it is not a Th, also you could use `Text` component and put it as `div` so you won't need to define `font-size`, `font-weight`, etc.

Reply:
I cant because Text doesnt allow onClick prop :/